src/ui/components/layout/Body.tsx

```diff
@@ -2,14 +2,18 @@ import { PropsWithChildren } from 'react';
 import styled from '@emotion/styled';
 import styleToken from '../../styles/styleToken.css';
 
-export default function Body({ children }: PropsWithChildren) {
-  return <Container>{children}</Container>;
+type BodyProps = {
+  [K: string]: unknown;
+};
+
+export default function Body({ children, ...props }: PropsWithChildren & BodyProps) {
```

Comment:
```suggestion
export default function Body({ children, ...props }: PropsWithChildren<BodyProps>) {
```

P2:
PropsWithChildren 활용 해주실때는 제네릭으로 추가되는 타입을 명시해주시면 됩니다!